documentation/environment-variables.md

```diff
@@ -20,3 +20,33 @@ For deployment in production, there needs to be a way to change variables in the
 
 To check what variables have been set in the app, open the console of your browser and enter:
 `window.__SERVER_CONFIG__`
+
+#### End-to-End Testing
+
+Since run-time variables are used to maintain the current feature set and can be changed, tests should be run against both versions of a specific feature set. In order to do this, the "feature flags" must be overridden in the test specifications.
```

Comment:
any specific reason to have feature flags in quotes?
```suggestion
Since run-time variables are used to maintain the current feature set and can be changed, tests should be run against both versions of a specific feature set. In order to do this, the feature flags must be overridden in the test specifications.
```